sources/packages/backend/apps/workers/src/controllers/assessment/assessment.controller.ts

```diff
@@ -442,7 +448,11 @@ export class AssessmentController {
         jobLogger.log(
           `The assessment calculation order has been verified and the assessment id ${assessmentId} is ready to be processed.`,
         );
-        this.createOutputForProgramYearTotals(programYearTotalAwards, result);
+        this.createOutputForProgramYearTotals(
+          programYearAwardsContributionTotal.awardTotal,
+          programYearAwardsContributionTotal.ftProgramYearContributionTotal,
```

Comment:
It still uses the "ft" here. Can we change it as we talked?

Reply:
The variable can be renamed to `contributionTotals` only in the same way `awardTotal` can be `awardTotals`.